lib/assessment.rb

```diff
@@ -13,14 +14,18 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.reduce {0, :+}
```

Comment:
You're right that `.reduce` is the right tool, but your syntax is off.
`enum.reduce(&:+)`